node/src/daemon/setup_reporter.rs

```diff
@@ -2052,22 +2050,11 @@ mod tests {
         let _guard = EnvironmentGuard::new();
         let base_dir = ensure_node_home_directory_exists(
             "setup_reporter",
-            "get_modified_setup_tilde_in_data_directory",
+            "get_modified_setup_tilde_in_config_file_path",
```

Comment:
Can you think up a more neutral name for this test without the explicit reference to the config-file (only)?